lms/product/plugin/misc.py

```diff
@@ -39,3 +39,25 @@ def is_assignment_gradable(self, lti_params: LTIParams) -> bool:
     def get_ltia_aud_claim(self, lti_registration: LTIRegistration) -> str:
         """Get the value of the `aud` claim used in LTI advantage requests."""
         return lti_registration.token_url
+
+    def get_document_url(self, request):
+        """Get a document URL from an assignment launch."""
+
+        # For assignments that don't use deep linking the source of truth for this information is our DB.
```

Comment:
```suggestion
        # Get the document URL from the assignment in the DB if we have one
```
I find it a bit weird to state this in terms of what it's not. I suspect this is due to this being introduced after deep linking, so deep linking was seen as the default?